gem/lib/pupperware/spec_helper.rb

```diff
@@ -94,29 +94,33 @@ def curl(hostname, port, endpoint)
   # Docker Compose Helpers
   ######################################################################
 
-  def docker_compose(command_and_args)
+  def docker_compose(command_and_args, stream: StringIO.new)
     overrides = IS_WINDOWS ?
                   'docker-compose.windows.yml' :
                   'docker-compose.override.yml'
     # Only use overrides file if it exists
     file_arg = File.file?(overrides) ? "--file #{overrides}" : ''
     run_command("docker-compose --file docker-compose.yml #{file_arg} \
                                 --no-ansi \
-                                #{command_and_args}", stream: StringIO.new)
+                                #{command_and_args}", stream: stream)
   end
 
   def docker_compose_up()
-    docker_compose('up --detach')
-    docker_compose('images')
+    docker_compose('up --detach', stream: STDOUT)
```

Comment:
In general, when do we want to supply a stream here? Like for more debug output?

Not sure how I know when I would use this, but maybe that'd be obvious when I encounter it?

Reply:
Yeah, I just wanted to get something in there to fix the immediate problem I introduced with the refactor.

We could always have separate methods like `docker_compose` and `capture_docker_compose` or something like that?  I don't know that that's any better though TBH.

Reply:
What does passing in a stream do again?

Maybe add docs to `docker_compose` or something.

Not against the change, just want to understand it

Reply:
Like it looks like you changed every call to `docker_compose`, which makes me think we'd want to do this literally everywhere in every repo?